test/unit-tests/function/matrix/ifft.test.js

```diff
@@ -0,0 +1,52 @@
+// test ifft
+
+import approx from '../../../../tools/approx.js'
+import math from '../../../../src/defaultInstance.js'
+import { arraySize } from '../../../../src/utils/array.js'
+import { isMatrix } from '../../../../src/utils/is.js'
+const ifft = math.ifft
+
+function idft (arr) {
+  if (isMatrix(arr)) return math.matrix(idft(arr.valueOf()), arr.storage())
+  const size = arraySize(arr)
+  return math.dotDivide(
+    math.matrixFromFunction(size, n => {
+      let acc = 0
+      math.matrix(arr).forEach((x, k) => {
+        acc = math.add(acc,
+          math.multiply(
+            math.exp(
+              math.multiply(
+                math.multiply(math.i, math.tau),
+                math.dot(n, math.dotDivide(k, size))
+              )
+            ),
+            x
+          )
+        )
+      })
+      return acc
+    }),
+    size.reduce((acc, n) => acc * n, 1)
+  )
+}
+
+describe('inv', function () {
```

Comment:
Same feedback points apply as those on the unit tests of `fft`

Reply:
Done.